### DISCUSSION TITLE
feature_currency

### DISCUSSION THREAD

**Discussion by AlexandraKoshur** [ryanhtafesa/T24S2T1EXTG04-REV3#4] [General]
@ryanhtafesa 

Hi Ryan,

I needed to change my calculator's name because it did not much the initial requirements and I lost a point in my individual submission for it. Now, instead of Currency Converter, it's Foreign Exchange Calculator. I changed the button name and the click event name accordingly. It is now Button x:Name="btnForeignExchange" Content="Foreign Exchange" Click="btnForeignExchange_Click"

Just wanted to let you know

**Comment by ryanhtafesa** [comment]
Thanks' for letting me know!
@AlexandraKoshur
